### DISCUSSION TITLE
Preview post option not appearing

### DISCUSSION THREAD

**Discussion by sid-r-singh** [estruyf/vscode-front-matter#165] [Q&A] [answered]
Hi there. Thanks for creating this. I used this extension with a Hugo website (`hugo v0.88.1+extended linux/amd64 BuildDate=2021-09-04T13:52:13Z`). Why is there no option to preview the post (as it appears in the documentaion) ?

## My system:
![image](https://user-images.githubusercontent.com/68782815/139523585-3891b1ea-83bd-4c7e-a4a2-34d775d4a9c9.png)

## As per the documentaion:
![image](https://user-images.githubusercontent.com/68782815/139523679-330063f9-b8fa-4f8f-be03-ad12f0f1e2e4.png)

Initially I thought that I must start the server to get this option. But even after running `hugo serve`, the 'open preview' option did not appear. What can I do to get it ?

**Comment by estruyf** [comment, marked as answer]
Only the `frontMatter.preview.host` setting should be required. Can you try to restart the VS Code instance, maybe something didn't come through correctly?

**Reply by sid-r-singh** [reply]
I did set `frontMatter.preview.host` to `http://localhost:1313`. Tried restarting whole system, not only VS Code, but no change. Tried in 2 different OSs (Win, Linux - Ubuntu) still no change.

![image](https://user-images.githubusercontent.com/68782815/139573589-9ce0311a-c976-4b32-b74e-7b5d4fc55133.png)

I have also added my content folder by right clicking -> add folder. So the dashboard is appearing correctly. Only this preview option is not.

![image](https://user-images.githubusercontent.com/68782815/139573664-0e1206d9-a755-439b-9ff1-c9bc52c34ab8.png)

**Reply by estruyf** [reply]
@sid-r-singh I have opened an issue for this. I will try out a couple of things today and hope to reproduce the issue. You can follow it up here: https://github.com/estruyf/vscode-front-matter/issues/166
